Anvil/Tools/Remote.pm

```diff
@@ -477,6 +483,7 @@ sub call
 					user       => $remote_user,
 					port       => $port, 
 					batch_mode => 1,
+					@$ossh_opts,
```

Comment:
This is technically correct, but generally we use '@{$ossh_opts}' to make the variable with the array reference easier to see.